scripts/extract_pdf_text.py

```diff
@@ -0,0 +1,22 @@
+import fitz
+import sys
+import argparse
+
+def extract_pdf_text(pdf_path: str, output_type: str = "text" ):
+    """ Extracts text from a PDF file and writes it to a text file. """
```

Comment:
where is this writing the text to? i see `sys.stdout`, but that's not a text file, right?

Reply:
it's writing to stdout buffer, b/c you're reading the output from stdin. Since you're the consumer of this script, feel free to make any requirements you want. If you'd rather refactor to change how we read pdfs, I can get on it.